examples/marathon-pace/__tests__/index.js

```diff
@@ -0,0 +1,8 @@
+it('should match the image snapshot foo', async () => {
+    await d3fc.loadExample(module);
+    const image = await page.screenshot({
```

Comment:
Does this not need to wait for the data to load?

Reply:
fixed 👍